acquia_cms.profile

```diff
@@ -40,6 +40,8 @@ function acquia_cms_install_tasks_alter(array &$tasks) {
  */
 function acquia_cms_install_tasks(): array {
   $tasks = [];
+  $start_time = date('h:i:s a', time());
+  \Drupal::state()->set('install_start_time', $start_time);
```

Comment:
@panshulK We can't add logic code directly here because this function gets invoked when each task runs.

Reply:
Hey @vishalkhode1 
Moved this logic to the `pre_start` function.